src/main/webapp/features/screens/game/dashboard/resourcesPanel/ct-resources-panel.js

```diff
@@ -9,9 +9,9 @@ angular.module('catan')
                 },
                 templateUrl: "/features/screens/game/dashboard/resourcesPanel/ct-resources-panel.html",
                 link: function(scope) {
-                    var currentGameUser = scope.game.getCurrentUser();
-                    if (currentGameUser) {
-                        scope.resources = currentGameUser.resources;
```

Comment:
resources wasn't updated on the view when they are changed. It is simplest way that I found to make them updatable